### PR TITLE
Bidmachine: Update JSON Schema Reference To Type String

### PR DESCRIPTION
**Pull request by agilfix**
Remove definitions object from schema and define types and other parameters directly in properties objects to ensure compatibility with more downstream systems that use this schema.

### CODE REVIEW

**Comment by SyntaxNode** [comment]
We'll assign PBS reviewers once you have approval from Bidmachine.

**Comment by hhhjort** [comment]
Both forms seem to be valid. As Scott says, we would want approval from Bidmachine (@meeeeeepo ?) before making such a change.

**Review by meeeeeepo — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]

**Review by SyntaxNode — APPROVED** [review]

**Pull request merged**
